pages/parent-enrollment.tsx

```diff
@@ -0,0 +1,125 @@
+import Wrapper from "@components/SDCWrapper";
+import {
+    Link,
+    Box,
+    Button,
+    Stack,
+    Center,
+    Text,
+    useRadio,
+    useRadioGroup,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CloseIcon } from "@chakra-ui/icons";
+
+function RadioCard(props) {
+    const { getInputProps, getCheckboxProps } = useRadio(props);
+
+    const input = getInputProps();
+    const checkbox = getCheckboxProps();
+    console.log(checkbox);
+    // Returns one radio card with the child's name. Button changes style when selected
+    return (
+        <Box as="label">
+            <input {...input} />
+            <Box
+                {...checkbox}
+                lineHeight="24px"
+                fontSize="16px"
+                fontWeight="normal"
+                textColor="#0C53A0"
+                width="340px"
+                height="50px"
+                border="2px solid #E1E1E1"
+                borderRadius="6px"
+                _checked={{
+                    fontWeight: "normal",
+                    bg: "white",
+                    textColor: "#0C53A0",
+                    border: "2px solid #0C53A0",
+                }}
+                textAlign="center"
+                py="13px"
+                my="10px"
+            >
+                {props.children}
+            </Box>
+        </Box>
+    );
+}
+
+export default function ParentEnrollClass(): JSX.Element {
+    // Next button is disabled by default, activates when a child is selected
+    const [isDisabled, setIsDisabled] = useState<boolean>(true);
+
+    function ChildrenButtons() {
+        // children to be retrieved from parent enroll endpoint
+        const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+
+        //when a radio button in the group is selected, Next button is activated
+        const { getRootProps, getRadioProps } = useRadioGroup({
+            onChange: () => {
+                setIsDisabled(false);
+                console.log("stuff changed");
+            },
+        });
+
+        const group = getRootProps();
+
+        // creates a group of radio buttons using the RadioCard component
+        return (
+            <Stack {...group}>
+                {children.map((value) => {
+                    const radio = getRadioProps({ value });
+                    return (
+                        <RadioCard key={value} {...radio}>
+                            {value}
+                        </RadioCard>
+                    );
+                })}
+            </Stack>
+        );
+    }
+
+    return (
+        <Wrapper>
+            <Link href="/" ml={"100%"}>
```

Comment:
Re: your question in the PR description - Instead of doing `ml={"100%"}` in the link, try wrapping it in a `Flex` with the `alignItems` property set to `"right"`

Reply:
:( the `alignItems` didn't work, but I did `justifyContent = "flex-end"` instead and it seemed to work